streams/readable-streams/general.js

```diff
@@ -32,6 +35,7 @@ test(() => {
     'constructor should throw when the type is empty string');
   assert_throws(new RangeError(), () => new ReadableStream({ type: 'asdf' }),
     'constructor should throw when the type is asdf');
+  assert_throws(error1, () => new ReadableStream({ type: { get toString() {throw error1;} } }), 'constructor should throw when ToString() throws');
```

Comment:
Both tests would be valid; to call a method you need to get it first. But it'd be good to add both tests.